presto-main/src/test/java/com/facebook/presto/execution/TestPlannerWarnings.java

```diff
@@ -56,8 +55,12 @@ public void setUp()
     @AfterClass(alwaysRun = true)
     public void tearDown()
     {
-        closeAllRuntimeException(queryRunner);
-        queryRunner = null;
+        try {
+            queryRunner.close();
+        }
+        finally {
```

Comment:
`finally` is not really needed. If `close` throws the test will fail anyway and we won't care whether `queryRunner` is `null` or not.